src/Configuration/Definition/BaseExtractorConfigDefinition.php

```diff
@@ -107,6 +108,18 @@ protected function getSshNode(): ArrayNodeDefinition
             //->end();
         // @formatter:on
 
+        $node->validate()
+            ->ifTrue(function ($v) {
+                if ($v['enabled'] === false) {
+                    return false;
+                }
+                $requiredKeys = ['sshHost', 'keys'];
+                $actualKeys = array_keys($v);
+
+                return array_intersect($requiredKeys, $actualKeys) !== $requiredKeys;
+            })
+            ->thenInvalid('Nodes "sshHost" and "keys" are required.');
```

Comment:
Jen bych tady doplnil, kde jsou (parameters.ssh) a že se to kontroluje "when SSH tunnel is enabled".